tcms/testruns/forms.py

```diff
@@ -19,6 +20,10 @@ class Meta:
 
     manager = UserField()
     default_tester = UserField(required=False)
+    start_date = DateTimeField()
+    stop_date = DateTimeField()
+    planned_start = DateTimeField()
+    planned_stop = DateTimeField()
```

Comment:
All of these fields should be `required=False` because either:
1) They have a default value coming from the DB engine or
2) They are allowed to be blank

As-is the model form requires the values and that's why you had to modify some test data.  

Some of the failing tests are when editing TestRun objects, you can give it a try manually. Because the UI doesn't know anything about these new fields but the form still requires them it leads to failures.